### PR TITLE
Support client.use(..)

### PR DESCRIPTION
**Pull request by djmitche**
I don't have chrome installed on my dev host so I'm pushing this in hopes the test passes :)

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -66,6 +66,14 @@ export default class Client {
     }
   }
 
+  use(optionsUpdates) {
+    const options = {
```
Comment (eliperelman) [inline src/Client.js]: For consistency with the rest of the code, let's remove the trailing comma. We can also just shorten this to:

```js
return new this.contructor({ ...this.options, ...optionsUpdated });
```

Are we cool with this not being a deep merge? Any other authorized scopes that were pre-existing would be replaced, not merged.

File: README.md [hunk 2]
```diff
@@ -127,6 +127,13 @@ documentation. The methods always returns a _promise_ for the response JSON
 object as documented in the REST API documentation. If you have JavaScript Async Function
 support, you can also `await` these methods.
 
+If you need to create a client similar to a existing client, but with some
+options changed, use `client.use(options)`:
+
+```js
+queue.use({authorizedScopes: [..]}).createTask(..).then(..);
```
Comment (eliperelman) [inline README.md]: Let's break this up into a separate line per chain:

```js
queue
  .use({ authorizedScopes: [..] })
  .createTask(..)
  .then(..);
```

File: src/Client.js (hunk 1 above)
Comment (djmitche) [reply src/Client.js]: I think that's OK, but we should be explicit about it in the README

**Comment by djmitche** [comment]
Let's try that.  I'll reflect the README changes back to tc-client, too.

**Comment by eliperelman** [comment]
🎉
